src/components/forms/Combobox.tsx

```diff
@@ -284,6 +284,8 @@ interface EntryOptions<T = any> {
   label: string;
   value: string;
   labelFn?: (resource: T) => string | JSX.Element;
+  formatLabel?: (resource: T) => string;
```

Comment:
Is `labelFn` not enough here or?

Reply:
@beganovich Hmm, I do think so. The `labelFn` is good as it returns a `JSX.Element`, which is very useful for formatting labels and options in the dropdown. However, the issue is that this function doesn't fit well in the following context:

`displayValue={(entry: Nullable<Entry>) => entry?.label ?? ''}`

`displayValue` property only accepts string but NOT the JSX.Element.

I think you used the `labelFn` for formatting options in the `ProductSelector`, but you still used `product_key` as the label when it is selected. In this case using `rate` or `name` properties will give us bad UX because I think if users select some kind of option value they will expect to see it in selected section.

Let me know your thoughts.